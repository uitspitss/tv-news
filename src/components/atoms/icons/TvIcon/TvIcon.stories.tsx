import React from 'react';
import { Story, Meta } from '@storybook/react/types-6-0';
import base from 'paths.macro';

import { TvIcon, TvIconProps } from './TvIcon';

const path = base.replace(
  /^.+\/(atoms|molecules|organisms|templates|pages)\/(.+)\//,
  '$1/$2',
);

export default {
  title: `${path}`,
  component: TvIcon,
} as Meta;

const Template: Story<TvIconProps> = (args) => <TvIcon {...args} />;

export const Default = Template.bind({});
