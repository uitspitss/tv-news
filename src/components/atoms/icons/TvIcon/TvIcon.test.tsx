import React from 'react';
import { render } from '@testing-library/react';

import { TvIcon } from './TvIcon';

describe('atoms/icons/TvIcon', () => {
  test('snapshot test', () => {
    const { container } = render(<TvIcon />);
    expect(container).toMatchSnapshot();
  });

  describe('Behavior', () => {
    beforeEach(() => {
      jest.clearAllMocks();
      render(<TvIcon />);
    });

    test.todo('write component test');
  });
});
