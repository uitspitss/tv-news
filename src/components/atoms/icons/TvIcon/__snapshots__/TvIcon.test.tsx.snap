// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`atoms/icons/TvIcon snapshot test 1`] = `
<div>
  <svg
    class="emotion-0 emotion-1"
    fill="none"
    height="24"
    stroke="currentColor"
    viewBox="0 0 24 24"
    width="24"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M9.75 17L9 20l-1 1h8l-1-1-.75-3M3 13h18M5 17h14a2 2 0 002-2V5a2 2 0 00-2-2H5a2 2 0 00-2 2v10a2 2 0 002 2z"
      stroke-linecap="round"
      stroke-linejoin="round"
      stroke-width="2"
    />
  </svg>
</div>
`;
